src/components/Benchmarks/BenchmarkInfo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Link from '@docusaurus/Link'
+import Translate from '@docusaurus/Translate'
+
+const BenchmarkInfo = (props) => {
+  return (
+    <div className="box">
```

Comment:
boring thing, but we need to search for a CSS class here first 
https://infima.dev/docs/utilities/shadows


```suggestion
    <div className="box shadow--md">
```